Dockerfile

```diff
@@ -4,7 +4,7 @@ ADD . /usr/src/sriov-network-device-plugin
 
 ENV HTTP_PROXY $http_proxy
 ENV HTTPS_PROXY $https_proxy
-RUN apk add --update --virtual build-dependencies build-base linux-headers && \
+RUN apk add --update --virtual build-dependencies build-base linux-headers make && \
```

Comment:
Do we need explicitly specify `make`?

Reply:
for ppc it is needed

Reply:
So Dockerfile.ppc64le is for building ppc64le image, this one would be for amd64 and should not require `make`?

Reply:
I agree